Does not work with browserify

When running browserify to bundle test files using chai, I am getting the following error:

```
Cannot find module './lib-cov/type'
```

This is because index.js has a conditional require that is not supported by browserify. There was a similar problem in chai itself that has been fixed, is it possible to apply the same fix to type-detect too? See chaijs/chai#28 and chaijs/deep-eql#1.

--- a/index.js
+++ b/index.js
@@ -1,3 +1,1 @@
-module.exports = process.env.type_COV
-  ? require('./lib-cov/type')
-  : require('./lib/type');
+module.exports = require('./lib/type');

--- a/test/bootstrap/index.js
+++ b/test/bootstrap/index.js
@@ -1,2 +1,4 @@
 global.assert = require('simple-assert');
-global.type = require('../..');
+global.type = process.env.type_COV
+  ? require('../../lib-cov/type')
+  : require('../../lib/type');

